bulkredditdownloader/file_name_formatter.py

```diff
@@ -89,7 +89,14 @@ def format_resource_paths(self, resources: list[Resource],
     def validate_string(test_string: str) -> bool:
         if not test_string:
             return False
-        return any([f'{{{key}}}' in test_string.lower() for key in FileNameFormatter.key_terms])
+        result = any([f'{{{key}}}' in test_string.lower() for key in FileNameFormatter.key_terms])
+        if result:
+            if 'POSTID' not in test_string:
+                logger.warning(
+                    f'Post ID not included in this file scheme, so file names are not guaranteed to be unique')
```

Comment:
"File names are not guaranteed to be unique as Post IDs are not included in the file scheme. Some files might not be downloaded due to name conflicts."

Reply:
@Serene-Arc